### PR TITLE
Merge `m` into `cggmp24/m`

### PR DESCRIPTION
**Pull request by survived**
Sync `cggmp24/m` branch with `m`

I additionally pushed some commits after the merge commit to address that CI was not working on `cggmp24/m` branch, and consequentially we had some workflows failing

### CODE REVIEW

**Comment by survived** [comment]
@maurges I did some other changes in PR other than merging, see https://github.com/LFDT-Lockness/cggmp21/pull/126/files/c66918583665b489bbffd9b1df51ed9279cd17aa..d506fc2b392acf6a0ea04719b0ef0b465f7ae31f

**Comment by survived** [comment]
Ah sorry, CI is still broken, it appeared fixed at a time. Pushed new commit, hopefully that will do it

**Comment by survived** [comment]
Yep it finally worked

File: .github/workflows/deps.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Deps tracker
 
 on:
   pull_request:
-    branches: [ "*" ]
+    branches: [ "**" ]
```
Comment (maurges) [inline .github/workflows/deps.yml:5]: Is it required because of a slash in the branch name? This is silly

**Review by maurges — APPROVED** [review]

File: .github/workflows/deps.yml (hunk 1 above)
Comment (survived) [reply .github/workflows/deps.yml:5]: Yep exactly

**Pull request merged**